extension/README.md.args.mjs

```diff
@@ -196,10 +196,12 @@ Your \`Staker UI\` tab should be almost done and working at this point.
 
 📦 Run \`yarn vercel\` to package up your frontend and deploy.
 
-> Follow the steps to deploy to Vercel. Once you log in (email, github, etc), the default options should work. It'll give you a public URL.
+> You might need to log in to Vercel first by running \`yarn vercel:login\`.
```

Comment:
is it intentional that you removed `Once you log in (email, GitHub, etc), the default options should work.` here?